core/themes/basis/css/component/backdrop-form.css

```diff
@@ -89,7 +89,9 @@ td.checkbox .form-item input {
   border-top: 0;
   padding: 0 0.125em;
 }
-
+details.filter-wrapper {
```

Comment:
I don't think this is needed anymore, since we aren't using `details` for a filter wrapper.